pkg/util/metric/m_register.go

```diff
@@ -14,8 +14,14 @@
 
 package metric
 
+import prom "github.com/prometheus/client_golang/prometheus"
+
+const constTenantKey = "tenant_id"
```

Comment:
We should just use `account` instead `tenant_id`, with value `GetDefaultTenant()`